### ISSUE TITLE
[FEATURE-REQUEST] Integratie BOR-MELD in Signalen

### ISSUE THREAD

**Issue by EricOosterom** [Signalen/product-steering#74]
CROW en Stichting RIONED i.s.m. gemeenten Breda, Delft, Amsterdam, Rotterdam, Utrecht, Den Haag, Almere, Roosendaal, Schiedam, Capelle aan den IJssel, Helmond, Barendrecht, Albrandswaard en Ridderkerk
Eric Oosterom
06-49854457
eric.oosterom@rioned.org

### Omschrijving / Summary
Op korte termijn integratie van BOR-MELD in Signalen

### Probleem beschrijving / Problem description
BRO-MELD is de uniforme registratie- en uitwisselmethodiek voor meldingen Beheer Openbare Ruimte (BOR). Zie https://www.crow.nl/thema-s/management-openbare-ruimte/beeldkwaliteit/bor-meld. Integratie van BOR-MELD in Signalen zou een belangrijke win-win zijn, want gemeentelijke assetmanagers hebben grote behoefte aan gestructureerde vastlegging van BOR-meldingen als belangrijke bron van informatie, terwijl integratie ook betere afhandeling en dus dienstverlening aan de samenleving zal betekenen. Beheerders zullen daarmee intern ook aanjager worden van implementatie van Signalen. En de gezamenlijke data- en kennisopbouw zal maatschappelijk tot aanzienlijke besparingen en kwaliteitsverbeteringen leiden. CROW en Stichting RIONED willen graag in overleg met de Signalen community om de integratie op de (ontwikkel)agenda te krijgen.

### Gebruikers criteria / User criteria
??? Een flinke groep gemeentelijke beheerders (zie selectie hierboven) steunt de BOR-MELD aanpak en de verbinding met Signalen.

### Consequentie nul-optie / Cost of not doing this request
Gebrek aan (beheer)informatie, bestaande verkokering en inefficientie blijven, kansen voor datagedreven werken blijven onbenut.

### Acceptatie criteria / Acceptance criteria
??? Is aan de Signalen-community dit in te vullen.

### Details feature / details of feature
??? Is aan de Signalen-community dit in te vullen.

### Urgentie / Priority
? Wat ons betreft hoog!

**Comment by ericherman** [comment]
@EricOosterom this looks interesting. As you mention Amsterdam is a part of Stichting RIONED, have you been in touch with @arvidsmits or others in the Amsterdam?

**Comment by EricOosterom** [comment]
@ericherman Nee, nog niet. Maar graag! @arvidsmits mag mij benaderen via eric.oosterom@rioned.org en 06-49854457. En ik zal ook contact proberen te zoeken.
